code/api/src/modules/product/types.js

```diff
@@ -14,6 +14,8 @@ const ProductType = new GraphQLObjectType({
     gender: { type: GraphQLInt },
     description: { type: GraphQLString },
     image: { type: GraphQLString },
+    style: { type: GraphQLString},
+    isSurvey: { type: GraphQLBoolean},
     createdAt: { type: GraphQLString },
     updatedAt: { type: GraphQLString }
```

Comment:
Have to add it to the fields.
Strange when we add it to mutations, it broke graphiql so have to play around with it